.github/workflows/ci.yml

```diff
@@ -0,0 +1,77 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - 'v*'
+      - 2.x
+      - /^\d+\.\d+\.\d+/
```

Comment:
maybe we should standardize on one way to name our version branches 😅

Reply:
`/^\d+\.\d+\.\d+/` is for the release tags though - for ember-test-selectors we use `'v*'` but we're not using the prefix here yet. I copied it anyway as we might want to streamline the tags across our repositories. `2.x` is mainly relevant now as `master` contains all the changes for `3.0` already.

Reply:
then it should be under `tags`, not `branches` 🤔

Reply:
You're right - we don't need this at all actually. I also renamed the `2.x` branch to `v2`.